packages/core/content-type-builder/server/services/components.js

```diff
@@ -20,7 +20,7 @@ const formatComponent = component => {
     apiId: modelName,
     schema: {
       icon: _.get(info, 'icon'),
-      name: _.get(info, 'name') || _.upperFirst(pluralize(uid)),
+      displayName: _.get(info, 'displayName') || _.upperFirst(pluralize(uid)),
```

Comment:
if displayName is required can we remove the second operand of the OR ? removing pluralize call as much as possible